changelogs/8.10.asciidoc

```diff
@@ -20,6 +20,7 @@ https://github.com/elastic/apm-server/compare/8.9\...8.10[View commits]
 [float]
 ==== Bug fixes
 - Add back handling of `queue.*` config for libbeat outputs, such as logstash and kafka {pull}11534[11534]
+- Accept empty object for `transaction.dropped_spans_stats.duration` field {pull}11117[11117]
```

Comment:
```suggestion
- Fix panic on missing `transaction.dropped_spans_stats.duration` field {pull}11117[11117]
```

nit, suggestion: Essentially the same meaning but may be more relevant to affected users

Reply:
yup, that sounds better